### PR TITLE
Small typos

### PR DESCRIPTION
**Pull request by lurch**
ping @naushir for confirmation.

### CODE REVIEW

File: documentation/asciidoc/accessories/ai-camera/details.adoc [hunk 1]
```diff
@@ -49,7 +49,7 @@ Once `libcamera` dequeues the image and inference data buffers from the kernel,
 | Floating point array storing the input tensor.
 
 | `CnnOutputTensorInfo`
-| Network specific parameters describing the output tensors structure:
+| Network specific parameters describing the output tensor's structure:
```
Comment (naushir) [inline documentation/asciidoc/accessories/ai-camera/details.adoc:52]: I think this and below is meant to be plural.

Comment (lurch) [reply documentation/asciidoc/accessories/ai-camera/details.adoc:52]: The reason I thought it was meant to be singular, is that `CnnOutputTensor` just above is described as "Floating point array storing the output tensor." and `CnnInputTensor` is described as "Floating point array storing the input tensor." ?

Comment (naushir) [reply documentation/asciidoc/accessories/ai-camera/details.adoc:52]: That should probably be plural as well.  Input tensor is singular.

Comment (lurch) [reply documentation/asciidoc/accessories/ai-camera/details.adoc:52]: Ahhh - so a single input tensor, but multiple output tensors?

Comment (naushir) [reply documentation/asciidoc/accessories/ai-camera/details.adoc:52]: Yup, that's correct.

**Review by nathan-contino — APPROVED** [review]
LGTM!

**Pull request merged**